Angular/src/app/add-capability/add-capability.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="modal-header">
+    <h5 class="modal-title" id="exampleModalLabel">Add Capability</h5>
+    <button
+      type="button"
+      class="close"
+      aria-label="Close"
+      (click)="activeModal.dismiss()"
+    >
+      x
```

Comment:
I would suggest going through and neatening up the HTML. Like putting this > on the line above it and removing empty lines